shell/components/nav/Group.vue

```diff
@@ -69,7 +69,7 @@ export default {
         if (overviewRoute && grp.overview) {
           const route = this.$router.resolve(overviewRoute || {});
 
-          return this.$route.fullPath === route?.route?.fullPath;
+          return this.$route.fullPath.split('#')[0] === route?.route?.fullPath;
```

Comment:
I think this is the issue related to the navigation which includes anchors and query parameters. Maybe we should replace it with `include()` instead of exact match?
The issue is captured in the video.

Reply:
Specifically for the case of `https://rancher.cnotv.xyz/dashboard/c/local/explorer/members#cluster-membership`, this path is not recognized as it's not a full match and causes error looks like.

Reply:
This issue needs more investigation and has one open issue.